planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -318,7 +318,11 @@ AvoidancePlanningData AvoidanceModule::calcAvoidancePlanningData(DebugData & deb
 void AvoidanceModule::fillAvoidanceTargetObjects(
   AvoidancePlanningData & data, DebugData & debug) const
 {
-  const auto expanded_lanelets = utils::avoidance::getTargetLanelets(
+  using utils::avoidance::fillObjectStoppableJudge;
+  using utils::avoidance::filterTargetObjects;
+  using utils::avoidance::getTargetLanelets;
+
+  const auto expanded_lanelets = getTargetLanelets(
     planner_data_, data.current_lanelets, parameters_->detection_area_left_expand_dist,
     parameters_->detection_area_right_expand_dist * (-1.0));
 
```

Comment:
// Separate dynamic objects based on whether they are inside or outside of the expanded lanelets.
just a suggestion for adding comment

Reply:
Add comment line in https://github.com/autowarefoundation/autoware.universe/pull/3978/commits/0d099f4f4baaa6ff9134f77160612075a7b5c162.